MdiChild/MdiChild.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by voidThread on 12.02.18.
+//
+#pragma once
+
+#include "PaintViewWidget/PaintViewWidget.h"
+
+class MdiChild : public PaintViewWidget
```

Comment:
This is goind to be QDialog in the future. PaintViewWidget is just not enough, but "good enough for now".